src/gui/DatabaseWidget.cpp

```diff
@@ -1495,6 +1530,16 @@ bool DatabaseWidget::currentEntryHasTotp()
     return currentEntry->hasTotp();
 }
 
+bool DatabaseWidget::currentEntryHasIconSet()
```

Comment:
Technically this function is telling you if the current entry has a custom icon. I would actually prefer this function be in the Entry class itself. DatabaseWidget is overburdened as-is. The call chain should be:
```
auto entry = dbWidget->currentEntry();
if (entry && !entry->hasCustomIcon()) {
	// Do something amazing
}
```

Reply:
Actually this functions is not really needed. The icon checking is also made in the IconDownloader classes. This function is now removed.